docs/modules/models/llms/integrations/databricks.ipynb

```diff
@@ -0,0 +1,462 @@
+    "  * outputs: `[{\"type\": \"string\"}]`\n",
+    "\n",
+    "If the model signature is incompatible or you want to insert extra configs, you can set `transform_input_fn` and `transform_output_fn` accordingly."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {
+      "byteLimit": 2048000,
+      "rowLimit": 10000
+     },
+     "inputWidgets": {},
+     "nuid": "7496dc7a-8a1a-4ce6-9648-4f69ed25275b",
+     "showTitle": false,
+     "title": ""
+    }
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "'I am happy to hear that you are in good health and as always, you are appreciated.'"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# If running a Databricks notebook attached to an interactive cluster in \"single user\" \n",
+    "# or \"no isolation shared\" mode, you only need to specify the endpoint name to create \n",
+    "# a `DatabricksEndpoint` instance to query an serving endpoint in the same workspace.\n",
+    "llm = DatabricksEndpoint(endpoint_name=\"dolly\")\n",
+    "\n",
+    "llm(\"How are you?\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {
+      "byteLimit": 2048000,
+      "rowLimit": 10000
+     },
+     "inputWidgets": {},
+     "nuid": "0c86d952-4236-4a5e-bdac-cf4e3ccf3a16",
+     "showTitle": false,
+     "title": ""
+    }
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "'Good'"
+      ]
+     },
+     "execution_count": 34,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "llm(\"How are you?\", stop=[\".\"])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {
+      "byteLimit": 2048000,
+      "rowLimit": 10000
+     },
+     "inputWidgets": {},
+     "nuid": "5f2507a2-addd-431d-9da5-dc2ae33783f6",
+     "showTitle": false,
+     "title": ""
+    }
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "'I am fine. Thank you!'"
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Otherwise, you can manually specify the Databricks workspace hostname and personal access token \n",
+    "# or set `DATABRICKS_HOST` and `DATABRICKS_API_TOKEN` environment variables, respectively.\n",
+    "# We strongly recommend not exposing the API token explicitly inside a notebook.\n",
+    "# You can use Databricks secret manager to store your API token securely.\n",
+    "\n",
+    "import os\n",
+    "os.environ[\"DATABRICKS_API_TOKEN\"] = dbutils.secrets.get(\"myworkspace\", \"api_token\")\n",
```

Comment:
This doc doesn't mention how to install `dbutils`. Shall we use a link for how to get the token instead?